start-aws.py

```diff
@@ -0,0 +1,62 @@
+import os
+import time
+
+import boto.ec2
```

Comment:
Using both `boto` and `libcloud` in the same repository makes me sad. (Even though we already use both in flocker itself.